share/spice/indego_bikeshare_locations/indego_bikeshare_locations.js

```diff
@@ -0,0 +1,78 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_indego_bikeshare_locations = function(api_result) {
+        if (!api_result || !api_result.features) {
+            return Spice.failed('indego_bikeshare_locations');
+        }
+
+   
+
+        DDG.require('moment.js', function() {
+            DDG.require('maps', function() {
+             moment.locale('en', {
+                    relativeTime : {
+                        past:   "%s ago",
+                        s:    "a sec",
+                        ss:   "%d sec",
+                        m:    "a min",
+                        mm:   "%d min",
+                        h:    "an hour",
+                        hh:   "%d hours",
+                        d:    "a day",
+                        dd:   "%d days",
+                        M:    "a month",
+                        MM:   "%d months",
+                        y:    "a year",
+                        yy:   "%d years"
+                    }
+                });    
+            
+                Spice.add({
+                    id: 'indego_bikeshare_locations',
+                    name: 'Indego Bike Share',
```

Comment:
Let's change this to "Bike Sharing". That wall all these bike sharing IA's will have the same tab name.